doc/source/whatsnew/v1.0.0.rst

```diff
@@ -247,6 +247,7 @@ Missing
 MultiIndex
 ^^^^^^^^^^
 
+- Constructior for :class:`MultiIndex` verifies that the given ``sortorder`` is compatible with the actual ``lexsort_depth``  if ``verify_integrity`` parameter is ``True`` (the default)
```

Comment:
sorry to keep adding, but need an issue number here, it *can* be this PR number

Reply:
No worries, commit are cheap.